assets/src/scripts/components/hydrograph/graph-brush.js

```diff
@@ -70,11 +72,24 @@ export const drawGraphBrush = function(container, store) {
                 selection = [0, layout.width - layout.margin.right];
             }
             brushElem.remove();
+
             const group = svg.append('g').attr('class', 'brush')
                 .attr('transform', `translate(${layout.margin.left},${layout.margin.top})`);
+
             graphBrush.extent([[0, 0], [layout.width - layout.margin.right, layout.height - layout.margin.bottom]]);
+
+            // Creates the brush
             group.call(graphBrush);
+
+            // Fill & round corners of brush handles, set mouseover highlight
+            svg.selectAll('.handle').classed('brush-handle-fill', true)
+                .attr('rx',15).attr('ry',15)  // Round corners
+                .on('mouseover', function(){
```

Comment:
Since changing the class won't work, let's just go ahead and put the fill style in the code (i.e. remove fill: #345d96 from the style sheet). I also take back what I said about using USWDS color tokens. It doesn't look like these can be easily used to set the fill property. But the colors you picked are fine.